PowerShellTools/Classification/PowerShellTokenizationService.cs

```diff
@@ -88,6 +88,7 @@ private void UpdateTokenization()
                             NotifyOnTagsChanged(BufferProperties.Classifier, currentSnapshot);
                             NotifyOnTagsChanged(BufferProperties.ErrorTagger, currentSnapshot);
                             NotifyOnTagsChanged(typeof(PowerShellOutliningTagger).Name, currentSnapshot);
+			    NotifyBufferUpdated();

```

Comment:
This looks good in VS. Looks like Github usually messes up the indentation and spacing.